### ISSUE TITLE
MD Toolbar height weird

### ISSUE THREAD

**Issue by patrikx3** [angular/material2#3746]
Please check my site:
https://pages.corifeus.tk/

When you size the browser width with a toolbar, it is correct, instead about the middle width it becomes too small.

Like toolbar: 
width  ~1920 - toolbar height = X
width ~889 toolbar height = 0.8 * X
width ~785 toolbar height = X

It should be the same, except maybe when width < 600.

Widest, height correct:
![image](https://cloud.githubusercontent.com/assets/4770180/24267729/90212476-100b-11e7-993e-0a7bab745064.png)

Middle, height it too narrow:
![image](https://cloud.githubusercontent.com/assets/4770180/24267746/9fe0c484-100b-11e7-85a2-f39fa14f4219.png)

Narrow, height is correct:
![image](https://cloud.githubusercontent.com/assets/4770180/24267752/a77d7fde-100b-11e7-9feb-faac8f8ab9d7.png)

**Comment by patrikx3** [comment]
critical error:
```css
@media (max-width: 960px) and (orientation: landscape) {
 .mat-toolbar {
     min-height: 48px;
 }
}
```

```scss
// Set the default height for the toolbar.
@include mat-toolbar-height($mat-toolbar-height-desktop);
```

**Comment by willshowell** [comment]
You can see from #2157 that those numbers are coming from the [Material spec for app bars](https://material.io/guidelines/layout/structure.html#structure-app-bar).
> Default heights:
> 
> Mobile Landscape: 48dp
> Mobile Portrait: 56dp
> Tablet/Desktop: 64dp

**Comment by patrikx3** [comment]
ciao, chrome debian is not mobile landscape. still it changes the toolbar height.

**Comment by patrikx3** [comment]
sent you 3 screenshots. they all desktop landscape. still it shrinks like it is a mobile landscape.

**Comment by patrikx3** [comment]
sorry i sent  you mobile screenshot, i send you a new 3 shots. same bug. sorry.

**Comment by patrikx3** [comment]
Still it is the first one is once like this height, then smaller, then the first again, werid!!! 
CIAO!

Full HD, toolbar as expected:
![image](https://cloud.githubusercontent.com/assets/4770180/24298125/590d1248-10a5-11e7-804c-3a8d24b04554.png)

Resize browser, about middle , it is wrong, too small:
![image](https://cloud.githubusercontent.com/assets/4770180/24298170/7c3b1cba-10a5-11e7-9939-a299252c1177.png)

I bit smaller resize, now the toolbar height is correct:
![image](https://cloud.githubusercontent.com/assets/4770180/24298203/910ad98c-10a5-11e7-8412-a84792334b97.png)

**Comment by willshowell** [comment]
Ahh it looks like this might have been fixed by #3312, which will be included in the next release.

**Comment by patrikx3** [comment]
cool, thanks so much.
